packages/types/src/indexed-db.ts

```diff
@@ -141,6 +141,8 @@ export interface IndexedDbInterface {
   getAuctionOutstandingReserves(
     auctionId: AuctionId,
   ): Promise<{ input: Value; output: Value } | undefined>;
+
+  hasNativeAssetBalance(): Promise<boolean>;
```

Comment:
Are `NativeAsset` and `StakingAsset` the same thing? I'm not sure, but it might make sense to use the same naming throughout so it doesn't cause confusion. (although for me personally, `NativeAsset` is the clear naming)

Reply:
yeah, good point. they're the same, but will stick with `StakingAsset` for consistency